src/routes/console/project-[project]/auth/security/updateUsersLimit.svelte

```diff
@@ -32,7 +32,7 @@
         submitting = true;
         try {
             await sdk.forConsole.projects.updateAuthLimit(projectId, isLimited ? newLimit : 0);
-            invalidate(Dependencies.PROJECT).then(() => (submitting = false));
+            await invalidate(Dependencies.PROJECT).then(() => (submitting = false));
```

Comment:
Why not put `submitting = false` after? Since we're using await anyways

Reply:
godo catch!